### PR TITLE
[topo]: Update t0-64 topology to have /26 network on Vlan1000

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

**Comment by lguohan** [comment]
@pavelsh, do you have assumption on the subnet size?

**Comment by pavel-shirshov** [comment]
I used /22 in the fast reboot test.
I think /21 is better for this patch.

File: ansible/templates/topo/t0-64.j2 [hunk 1]
```diff
@@ -61,7 +61,7 @@
             <RouteMapOut i:nil="true"/>
             <Vrf i:nil="true"/>
             <a:Name>BGPVac</a:Name>
-            <a:PeersRange>192.168.0.0/27</a:PeersRange>
+            <a:PeersRange>192.168.0.0/26</a:PeersRange>
```
Comment (pavel-shirshov) [inline ansible/templates/topo/t0-64.j2]: /21 is better. It allows to have 2048 hosts. 2000 is target for the fast reboot test

**Comment by stcheng** [comment]
@pavel-shirshov I've updated the pull request.

**Pull request merged**